feat(curriculum): add Artifact Provenance Auditor workshop (#68860)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-artifact-provenance-auditor/6a57c331ae94fa4631b6086b.md

---
id: 6a57c331ae94fa4631b6086b
title: Step 12
challengeType: 1
dashedName: step-12
---

# --description--

> Move the artifact with an `id` of `102` to `"Hall B"` in the year `2026`, then log that artifact's `locations`.

Reviewer: ```suggestion
Move the artifact with the `id` `102` to `"Hall B"` in the year `2026`. Then, log that artifact's `locations` array.
```
